src/NuGet.Core/NuGet.Commands/RestoreCommand/DependencyGraphResolver.cs

```diff
@@ -112,8 +117,7 @@ public DependencyGraphResolver(RestoreCollectorLogger logger, RestoreRequest res
                     // This is guaranteed to be computed before any graph with a RID, so we can assume this will return a value.
 
                     // PCL Projects with Supports have a runtime graph but no matching framework.
-                    var runtimeGraphPath = _request.Project.TargetFrameworks.
-                            FirstOrDefault(e => NuGetFramework.Comparer.Equals(e.FrameworkName, tfmNonRidGraph.Framework))?.RuntimeIdentifierGraphPath;
+                    var runtimeGraphPath = projectTargetFramework?.RuntimeIdentifierGraphPath;
```

Comment:
projectTargetFramework probably sohuldn't be null. if it's null, we'd likely have other issues.